recipes/bbf/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "bbf" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/bbf-{{ version }}.tar.gz
+  sha256: bfd4bb88b4281391e5693a8f7fb4a4c09a94f96712d0dd9d8c7c3c059e99c547
+
+build:
+  skip: true  # [py<39 or win]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - {{ stdlib("c") }}
+    - cmake >=3.18
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
```

Comment:
Please duplicate these two in host

Reply:
Thanks for your review @xhochy, I just added that